zad3/Gemfile

```diff
@@ -3,7 +3,7 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '2.5.1'
+ruby '2.3.1'
```

Comment:
RIP `ruby 2.5.1` ⚰️ 😢  Jeśli masz już `rvm` to żeby zainstalować nowego rubiego wystarczy:
```
rvm install 2.5.1
rvm use 2.5.1
```